scripts/markets/create_markets.js

```diff
@@ -107,6 +107,8 @@ async function run() {
   )
   let ammAddress = ret.logs[2].args["0"]
   ammWBTCUSDC = await MinterAmm.at(ammAddress)
+  console.log(`deployed WBTC AMM at address ${ammAddress}`)
```

Comment:
I added these so we can get the lpToken addresses for the 2 AMM's when deploying locally, which later we can add to the create_staking_contracts.js script